### PR TITLE
Updated deployment config. Fixed assembly plugin to work with newer v…

### PR DESCRIPTION
**Pull request by bibryam**
Updated deployment config to remove refs to latest tag.
Fixed assembly plugin to work with newer version of mvn.
Added IntelliJ files to gitignore

### CODE REVIEW

**Comment by cloudbotinmarsat** [comment]
Can one of the admins verify this patch?

**Comment by hammerdj1** [comment]
test this please

File: pom.xml [hunk 1]
```diff
@@ -350,9 +350,11 @@
 				<version>2.6</version>
 				<configuration>
 					<finalName>kubernetes</finalName>
-					<descriptorRefs>project</descriptorRefs>
+                    <descriptorRefs>
```
Comment (cloudbotinmarsat) [inline pom.xml]: MINOR Make this line start at column 11. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=xml%3AIndentCheck)

File: pom.xml [hunk 2]
```diff
@@ -350,9 +350,11 @@
...
+                        <descriptorRef>jar-with-dependencies</descriptorRef>
+                    </descriptorRefs>
 					<archiveBaseDirectory>kube/</archiveBaseDirectory>
-					<formats>zip</formats>
+					<formats><format>zip</format></formats>
```
Comment (cloudbotinmarsat) [inline pom.xml]: MINOR Make this line start at column 13. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=xml%3AIndentCheck)
MINOR Node should be on the next line [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=xml%3ANewlineCheck)
MINOR Missing newline after last element [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=xml%3ANewlineCheck)

**Comment by cloudbotinmarsat** [comment]
SonarQube analysis reported 4 issues
* MINOR 4 minor

Watch the comments in this conversation to review them.

**Pull request merged**